samples/v1/language_classify_gcs.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DO NOT EDIT! This is a generated sample ("Request",  "language_classify_gcs")
+
+// sample-metadata:
+//   title: Classify Content (GCS)
+//   description: Classifying Content in text file stored in Cloud Storage
+//   usage: node samples/v1/language_classify_gcs.js [--gcs_content_uri "gs://cloud-samples-data/language/classify-entertainment.txt"]
+
+'use strict';
+
+// [START language_classify_gcs]
+
+const {LanguageServiceClient} = require('@google-cloud/language').v1;
+
+/**
+ * Classifying Content in text file stored in Cloud Storage
+ *
+ * @param gcsContentUri {string} Google Cloud Storage URI where the file content is located.
+ * e.g. gs://[Your Bucket]/[Path to File]
+ * The text file must include at least 20 words.
+ */
+function sampleClassifyText(gcsContentUri) {
+  const client = new LanguageServiceClient();
+  // const gcsContentUri = 'gs://cloud-samples-data/language/classify-entertainment.txt';
+
+  // Available types: PLAIN_TEXT, HTML
+  const type = 'PLAIN_TEXT';
+
+  // Optional. If not specified, the language is automatically detected.
+  // For list of supported languages:
+  // https://cloud.google.com/natural-language/docs/languages
+  const language = 'en';
+  const document = {
+    gcsContentUri: gcsContentUri,
+    type: type,
+    language: language,
+  };
+  client
```

Comment:
We would like to lock to using async functions and `async`/`await` style here.  We cover this a bit in the style guide:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

The pattern here would look like:
```js
async function classify() {
  const [response] = await client.classifyText({document});
  for (const category of response.categories) {
    // do stuff
  }
}
classify();
```

The `catch` at the end is not needed either.

Reply:
agreed :+1: It's also worth noting that [top level await is coming](https://v8.dev/features/top-level-await).

![image](https://user-images.githubusercontent.com/194609/67425637-3f239080-f58d-11e9-9842-0580903566b0.png)